index.html

```diff
@@ -803,10 +803,10 @@ <h4 id="property-values-and-evaluation"><a class="toc-anchor" href="#property-va
 </ol>
 <h3 id="input-descriptors"><a class="toc-anchor" href="#input-descriptors">§</a> Input Descriptors</h3>
 <p><em>Input Descriptors</em> are objects used to describe the information a Verifier
-requires of a Holder before they will proceed with an interaction. <em>Input
-Descriptor Objects</em> contain a schema URI that links to the schema of the
-required input data, constraints on data values, and an explanation why a
-certain item or set of data is being requested:</p>
+requires of a Holder before they will proceed with an interaction. If no <code>submission_requirement</code> objects are presenet, to successfully fulfill a <code>Presentation Definition</code>, all <code>input_descriptor</code> objects <strong><strong>MUST</strong></strong> be satisfied.</p>
```

Comment:
```suggestion
requires of a Holder before they will proceed with an interaction. If no <code>submission_requirement</code> objects are present, all <code>input_descriptor</code> objects <strong><strong>MUST</strong></strong> be satisfied.</p>
```
I think this says about the same thing, but reads a bit clearer to me.

Reply:
much nicer! updated.